src/manage_sql_secinfo.c

```diff
@@ -171,6 +171,95 @@ increment_transaction_size (int* current_size)
     }
 }
 
+/**
+ * @brief Split a file.
+ *
+ * @param[in]  path  Path to file.
+ * @param[in]  size  Approx size of split files.
+ *
+ * @return Temp dir holding split files.
+ */
+static const gchar *
+split_xml_file (gchar *path, const gchar *size)
```

Comment:
Because the executed command contains some parts specific to the CPE dictionary, I think this should either be made more generic or renamed to clarify that it's for the CPEs.